lib/commons/aria/index.js

```diff
@@ -897,7 +897,7 @@ lookupTables.role = {
 		},
 		owned: null,
 		nameFrom: ['author', 'contents'],
-		context: ['treegrid', 'tree']
```

Comment:
It is, but treeitem doesn't require it as context:
https://www.w3.org/TR/wai-aria-1.1/#treeitem